tock/tock/static/js/components/filter_report_table.js

```diff
@@ -0,0 +1,46 @@
+(function () {
+  const unfiledTimecardFilter = document.getElementById('unfiled-timecards-org-filter-select');
+  const unfiledTimecardTable = document.getElementById('js-unfiled-timecards');
+  const rows = unfiledTimecardTable.getElementsByTagName('tr');
+
+  const showRow = (row) => {
+    row.style.display = 'table-row';
+  };
+
+  const hideRow = (row) => {
+    row.style.display = 'none';
+  };
+
+  const filterByOrg = (selectedOrg) => {
+    for (i in rows) {
+      // skip the header row, which contains `th`, and other gunk
```

Comment:
Random thought to take or leave as you see fit: What if we just selected tbody rows?

Reply:
That would work! There are a couple of artifacts at the end of `rows` that come from it being an `HTMLCollection`; I should probably strip them out instead of being lazy and letting the try/catch loop filter them out.